pkg/deploy/deploy_test.go

```diff
@@ -28,26 +31,27 @@ func TestGetParameters(t *testing.T) {
 			},
 		},
 		{
-			name: "valid",
+			name: "when all parameters present, everything is copied",
 			ps: map[string]interface{}{
 				"adminApiCaBundle":                   nil,
 				"databaseAccountName":                nil,
 				"extraClusterKeyvaultAccessPolicies": nil,
 			},
 			config: Configuration{
-				DatabaseAccountName:                "databaseAccountName",
-				ExtraClusterKeyvaultAccessPolicies: []interface{}{"a", 1},
+				DatabaseAccountName:                &databaseAccountName,
+				AdminAPICABundle:                   &adminApiCaBundle,
```

Comment:
Previously in this case we had no `AdminAPICABundle` in `Configuration`, but we expected `adminApiCaBundle` with empty value in `want`ed `arm.Parameters`.

We changed this behaviour. Do we need to review our configs to make sure that we do not rely on the old behaviour?

Reply:
@petrkotas please do this - do you have access to the RP config repo?  If not please ask one of the other EMEA team members.

Reply:
@m1kola thanks for the catch. Template params have interesting behavior. When you do not specify default values and parameter is not provided, you are asked to provide it.

I have fixed it with providing the default where required.